src/hash.cr

```diff
@@ -1611,42 +1611,45 @@ class Hash(K, V)
 
   # Returns a new hash with all keys converted using the block operation.
   # The block can change a type of keys.
+  # The block optionally yields the value as well
```

Comment:
It isn't mentioned previously that the block yields the key, so I think both should be mentioned:
```suggestion
  # The block yields key and value.
```